Doc/Tutorial/chapter_contributing.tex

```diff
@@ -4,13 +4,11 @@ \section{Bug Reports + Feature Requests}
 
 Getting feedback on the Biopython modules is very important to us. Open-source projects like this benefit greatly from feedback, bug-reports (and patches!) from a wide variety of contributors.
 
-The main forums for discussing feature requests and potential bugs are the
-\href{http://biopython.org/wiki/Mailing_lists}{Biopython mailing lists}:
+The main forum for discussing feature requests and potential bugs is the
+\href{http://biopython.org/wiki/Mailing_lists}{Biopython mailing list}:
 
 \begin{itemize}
   \item \mailto{biopython@biopython.org} -- An unmoderated list for discussion of anything to do with Biopython.
-
```

Comment:
I think we can remove this entire bullet point list (the ``itemize`` environment) now, it seems unnecessary for one entry only.

Reply:
I think we should remove both bullet point list entries. The Explicit mailing list e-mail address should be mentioned in the previous paragraph or as a footnote.

Reply:
OK